README.md

```diff
@@ -143,7 +143,9 @@ spec:
         alibabacloud.com/sgx_epc_MiB: 20
 ```
 
-If you want a remote attestation, you should mount `/var/run/aesmd/aesm.socket` in your container, maybe like this:
+If you want a remote attestation, you have two options:
+
+1. mount `/var/run/aesmd/aesm.socket` manually in your container, maybe like this:
```

Comment:
If aesm socket attachment is enabled in sgx-device-plugin, application container should not mount aesm.socket agiain.

Reply:
1. mount
=>
1. Mount